### ISSUE TITLE
Output with directly in line

### ISSUE THREAD

**Issue by dukaev** [DamirSvrtan/fasterer#57]
👋Hi
With this output, we can faster go to files directly in line.
I checked in [RubyMine](https://www.dropbox.com/s/qnza51d3haq885g/rubymine.mp4?dl=0) and [Sublime Text](https://www.dropbox.com/s/caxgsfhw5ftpxp4/sublime.mp4?dl=0), Visual Studio Code, also it works in vim.
Output:
```
code.rb:5 Using each_with_index is slower than while loop.
code.rb:21 Using each_with_index is slower than while loop.
code.rb:37 Enumerable#sort is slower than Enumerable#sort_by.
```

What do you think about this? If you like it I can do it :)

**Comment by DamirSvrtan** [comment]
Cool, sure thing, wanna make a PR?

**Comment by DamirSvrtan** [comment]
Would be great to support both types of outputting, for consistency sake. So that the output formatter can be configurable. I'm trying to think wether someone depends on the old output.

**Comment by dukaev** [comment]
I will make PR.
Output will configurable.
`.fasterer.yml`
```yaml
display:
	details-by-lines: true
```
**details-by-lines** will be **false** by default.

**Comment by dukaev** [comment]
🤔 Maybe?
```yaml
display: "by_lines"
```

**Comment by dukaev** [comment]
👋 @DamirSvrtan 
Can you please review my PR's?

**Comment by DamirSvrtan** [comment]
Merged this in with #64 !
